test/test_binary_ufuncs.py

```diff
@@ -88,6 +89,64 @@ def _make_tensor(shape, dtype, device, fill_ones=False) -> torch.Tensor:
 
 # TODO: update to use opinfos consistently
 class TestBinaryUfuncs(TestCase):
+    @ops(binary_ufuncs)
+    def test_broadcasting(self, device, dtype, op):
+        for sample in op.sample_inputs(device, dtype):
+            a = sample.input
+            b, *other_args = sample.args
+            if isinstance(b, torch.Tensor):
+                expected = torch.broadcast_shapes(a.shape, b.shape)
+            else:
+                expected = a.shape
+
+            actual = op(a, b, *other_args, **sample.kwargs).shape
+
+            msg = (
+                f"For {device} / {dtype}, torch.{op.name} broadcasts inputs of shapes {a.shape} and {b.shape} "
+                f"incorrectly: {actual} != {expected}"
+            )
+            torch.testing.assert_close(actual, expected, msg=msg)
+
+    @ops(binary_ufuncs)
+    def test_broadcast_python_scalar(self, device, dtype, op):
+        for shape in ((), (1,), (2,), (1, 2, 3)):
+            a = torch.ones(shape, device=device, dtype=dtype)
+            b_tensor = torch.ones((), device=device, dtype=dtype)
+            b_python = b_tensor.item()
```

Comment:
To use custom sizes in the future I suppose these tests will have to acquire the lhs and rhs make_tensor kwargs?

Reply:
I moved the `[lr]hs_make_tensor_kwargs` into `BinaryUfuncOpInfo` so they are accessible here.